doc/nrf/ug_tfm.rst

```diff
@@ -101,12 +101,29 @@ Logging
 TF-M employs two UART interfaces for logging: one for the secure part (MCUboot and TF-M), and one for the non-secure application.
 The logs arrive on different COM ports on the host PC.
 
-.. note::
-   * On the nRF5340 DK v1.0.0, you must connect specific wires on the kit to receive secure logs on the host PC.
-     Specifically, wire the pins **P0.25** and **P0.26** of the **P2** connector respectively to **RxD** and **TxD**  of the **P24** connector.
-     See :ref:`logging_cpunet` on the Working with nRF5340 DK page for more information.
-   * On the nRF5340 DK v2.0.0, there is one fewer COM port than on v1.0.0, so the secure and non-secure UART peripheral must be wired to the same pins.
-     Specifically, wire the pins **P0.25** and **P0.26** to **P0.20** and **P0.22**, respectively.
+
+Virtual COM ports in nRF5340 DK
+===============================
+On the nRF5340 DK v1.0.0, you must connect specific wires on the kit to receive secure logs on the host PC.
+Specifically, wire the pins **P0.25** and **P0.26** of the **P2** connector respectively to **RxD** and **TxD**  of the **P24** connector.
+See :ref:`logging_cpunet` on the Working with nRF5340 DK page for more information.
+
+On the nRF5340 DK v2.0.0, there are only two virtual COM ports available.
+By default, one of the ports is used by the non-secure UART peripheral from the application and the other one by the UART1 peripheral from the network core.
+
+There are several options to get UART output from the secure TF-M.
+
+* Disable the output for the network core.
+  For NCS samples, see :ref:`ug_multi_image_variables`.
```

Comment:
Do they need to configure a TF-M option that we have no Kconfig for?

If so they would need to be shown these docs, no?

https://docs.zephyrproject.org/3.0.0/guides/tfm/build.html#custom-cmake-arguments

Reply:
Well again depends on what is running on the network core, but should be realted to TF-M or its config options, but if you build the network core as child image then this points you how to set Kconfigs for a (network core)child image.

Reply:
Why are we linking to ug_multi_image_variables.

The CONFIG_TFM_UART1_RXD_PIN option is not a multi-image variable is it?

Reply:
It's not clear to the reader why he should read "ug_multi_image_variables" is it for changing TF-M pins or disabling network core logging?

Maybe something like:

* Disable the output for the network core and change the pins used by TF-M.
  The network core will usually have an NCS child image.
  To configure a child image see (replace with ref) https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/ug_nrf5340.rst#configuration-of-the-child-image.
  To configure logging in an NCS image see https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_logging.html#logging-in-ncs for the network.
  To change the pins used by TF-M the RXD (:kconfig:option:`CONFIG_TFM_UART1_RXD_PIN`) and TXD (:kconfig:option:`CONFIG_TFM_UART1_TXD_PIN`) pins of TF-M can be set to **P1.00** (32) and **P1.01** (33) **by Kconfig options in the application image**.

Reply:
> It's not clear to the reader why he should read "ug_multi_image_variables" is it for changing TF-M pins or disabling network core logging?
> 
> Maybe something like:
> 
> * Disable the output for the network core and change the pins used by TF-M.
>   The network core will usually have an NCS child image.
>   To configure a child image see (replace with ref) https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/ug_nrf5340.rst#configuration-of-the-child-image.
>   To configure logging in an NCS image see (change with ref) https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_logging.html#logging-in-ncs.
>   To change the pins used by TF-M the RXD (:kconfig:option:`CONFIG_TFM_UART1_RXD_PIN`) and TXD (:kconfig:option:`CONFIG_TFM_UART1_TXD_PIN`) Kconfig options in the application image can be set to **P1.00** (32) and **P1.01** (33).
-------
* Disable the output for the network core and change the pins used by TF-M.
  The network core will usually have an NCS child image.
  To configure a child image, see Configuration of the child image section described in :ref:`ug_nrf5340_multi_image`.
  To configure logging in an NCS image, see :ref:`ug_logging`.
  To change the pins used by TF-M, the RXD (:kconfig:option:`CONFIG_TFM_UART1_RXD_PIN`) and TXD (:kconfig:option:`CONFIG_TFM_UART1_TXD_PIN`) Kconfig options in the application image can be set to **P1.00** (32) and **P1.01** (33).

Another solution for child image config is to add label for the Configuration of the child image section, and reference it directly. Add e.g.
.. _ug_nrf5340_multi_image_child_image_config:

Configuration of the child image
================================